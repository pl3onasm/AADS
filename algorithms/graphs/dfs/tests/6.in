8

0 7
1 3
1 2
0 4
2 1
2 3
3 4
3 0
4 5
5 7
6 5
7 6
7 4


Source 
======
CLRS‚Å¥ Figure 20.5
s = 0
t = 1
u = 2
v = 3
w = 4
x = 5
y = 6
z = 7


