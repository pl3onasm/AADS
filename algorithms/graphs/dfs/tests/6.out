Discovery and finish times

   Node   Discovery    Finish
  label        time      time
      0           1        10
      1          11        16
      2          12        15
      3          13        14
      4           2         9
      5           3         8
      6           5         6
      7           4         7

Edge classification

  Starting   Ending      Edge
      node     node      type
         0        4         T
         0        7         F
         1        2         T
         1        3         F
         2        3         T
         2        1         B
         3        0         C
         3        4         C
         4        5         T
         5        7         T
         6        5         B
         7        4         B
         7        6         T